docs/how_to_use.md

```diff
@@ -75,13 +76,16 @@ You can also 'play' the window within the graph view. When playing the graph's c
 
 Some helpful **keyboard shortcuts** you may want to use are:
 
-*   'j' - To rewind
-*   'k' - To pause
-*   'l' - To Fast Forward
-*   'f' - To enlarge the window to full screen
+*   `'f'` - To enlarge the window to full screen
+*   `'j'` - To rewind
+*   `'k'` - To pause
+*   `'l'` - To Fast Forward
+*   `'m'` - Add a comment
 *   space bar - Toggle between play and pause
 *   left - Select the previous frame
 *   right - Select the next frame
+*   `CTRL+`, (Mac: `command+,`) - Jump to previous comment frame
+*   `CTRL+`. (Mac: `command+.`) - Jump to next comment frame
```

Comment:
Thanks very much! I should actually QC my commits that are based in previous QC...!!!